types/layui-src/index.d.ts

```diff
@@ -1,745 +1,18 @@
-// Type definitions for layui-src 2.5
-// Project: https://github.com/sentsin/layui
+// Type definitions for layui-src 2.6
+// Project: https://github.com/javabitar/layui
```

Comment:
Published package still points to original github project:
https://www.npmjs.com/package/layui
```bash
npm repo layui
```
why this change?